examples/aya-tool/myapp/Cargo.toml

```diff
@@ -5,14 +5,15 @@ edition = "2021"
 publish = false
 
 [dependencies]
-aya = "0.11"
+aya = { version = ">=0.11" }
```

Comment:
Please leave it unchanged, I think we should actually pin examples to minor versions.

Reply:
no problem.  i'm going to ditch this pr and send a new one. i forgot a few files and i have some major jet lag right now and goofed up my fork.  ugh